assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/rh-che.properties

```diff
@@ -1,24 +0,0 @@
-#
-# Copyright (c) 2016-2017 Red Hat, Inc.
-# All rights reserved. This program and the accompanying materials
-# are made available under the terms of the Eclipse Public License v1.0
-# which accompanies this distribution, and is available at
-# http://www.eclipse.org/legal/epl-v10.html
-#
-# Contributors:
-#   Red Hat, Inc. - initial API and implementation
-#
-
-# This is a routing suffix that can be used when building workspace agents external address
-# with a custom server evaluation strategy. This is similar to the che.docker.ip.external
-# property, but specific to the workspaces. So this is mainly useful when workspaces are not
-# hosted on the same machine as the main Che server and workspace master.
-# This is used as the default value if no other value is provided by the
-# `com.redhat.che.multitenant.Fabric8WorkspaceEnvironmentProvider.getUserCheTenantData()`
-# method. 
-che.fabric8.workspaces.routing_suffix=NULL
```

Comment:
Why this property has disappeared? How are we going to set [the agents routing suffix](https://github.com/redhat-developer/rh-che/blob/multi-tenant-rh-che/openshift/rh-che.config.yaml#L36)?

Reply:
It is not actual property for OpenShift SPI 6. There will fetch actual host from created routes. Is it OK or have I missed something?

Reply:
@davidfestal ?

Reply:
It's used in the server evaluation strategy used on OSIO (AlwaysExternal). And initially, it was also used to build the URLs of the workspace routes. But right, now it isn't anymore. Openshift decides what the workspace route hostname should be. So do you mean that, as a replacement for server evaluation strategy,  che6 retrieves this value directly from the routes ?
This I agree this wouldn't be necessary anymore.

Reply:
> Do do you mean that, as a replacement for server evaluation strategy, che6 retrieves this value directly from the routes?
@davidfestal Yes. It's implemented in che6 branch in this way

Reply:
OK

Reply:
👍